schemas/com.snowplowanalytics.snowplow/browser_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for browser contexts",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "browser_context",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "viewport": {
```

Comment:
Any risk that some of these might be nullable in a headless browser?

Reply:
On headless browsers there are some default values e.g. for pptr it is 800x600. I am not sure it could be made null since it is part of the CDP and can only be initiated with a number _(in normal circumstances)_.

We can be more lenient and make it nullable but not sure if we are being too cautious and allowing for exceptional cases to slip in the data and not the bad rows. WDYT ?

Reply:
Probably too cautious, let's leave as is I think.